pkg/network/setup/netstat_test.go

```diff
@@ -820,3 +820,9 @@ func newVMISpecMultusNetwork(name string) v1.Network {
 			}},
 	}
 }
+
+func newVMIStatusReadyIface(name string, IPs []string, mac, ifaceName string, infoSource string, queueCount int32) v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
I am commenting here because I cannot do so directly on the code which has not changed:

In the tests, I expect to see this used all over the place except `InfoSourceGuestAgent`.
But I see that it is not used in the SR-IOV case and `InfoSourceDomainAndGA`, which I think it should.

Reply:
I think I understand your point, but ... what good does this do for sriov interfaces ? 

Only reason I can see to do this, is for consistency - the attribute is useless for those interface types.

Can you confirm ? I guess I'm OK with making this consistent.

Reply:
Yes, for the SR-IOV it is indeed for consistency only, i.e. a default that says it is always the case.
But for the other one, it looks like a real bug.